activestorage/active.py

```diff
@@ -1,11 +1,16 @@
 import os
+import os.path
```

Comment:
no need for this import since os is imported above :grin:

Reply:
I'm sure this was necessary previously, since it's a separate module, but it does seem to work without this import in python 3.10. I'll remove.